### ISSUE TITLE
Make sure committees are up-to-date

### ISSUE THREAD

**Issue by morchickit** [hasadna/Open-Knesset#743]
Go to the Knesset committees page - http://main.knesset.gov.il/Activity/committees/Pages/default.aspx

- [ ] Make sure we have all committees in our DB

- [ ] Make sure we display all the current moshav member for a committee (since they seem to switch).

**Comment by morchickit** [comment]
@alonisser - help
הוספתי את הועדה לצדק חלוקתי, אבל אני לא מוצאת את מזהה הועודה בכנסת
מה לעשות?

**Comment by morchickit** [comment]
Also I believe that this committee - https://oknesset.org/committee/30/  actually changed name to this http://main.knesset.gov.il/Activity/committees/Urban/Pages/default.aspx @alonisser can you check it please? If so, i will add the new alias it has

**Comment by alonisser** [comment]
צדק חלוקתי 
http://online.knesset.gov.il/WsinternetSps/KnessetDataService/CommitteeScheduleData.svc/View_committee(610)
כלומר 610

**Comment by alonisser** [comment]
ועדה 30 אצלנו
מזהה כנסת שלה הוא 572
כלומר אפשר לראות אותה ב API 
http://online.knesset.gov.il/WsinternetSps/KnessetDataService/CommitteeScheduleData.svc/View_committee(572)
שם השם שלו הוא
"הוועדה המיוחדת לדיון בהצעת חוק להגברת התחרות ולצמצום הריכוזיות בשוק הבנקאות בישראל, התשע"ו-2016"

זה נשמע הגיוני?

**Comment by morchickit** [comment]
כן, היא פשוט פעם נקראה הועדה לחידוש עירוני
זה נוראי!!

**Comment by alonisser** [comment]
אגב מור אפשר די בקלות לדעתי לכתוב קוד שבודק שינויי שמות כאלו ומתקן אוטומטית
ומקים ועדות וכו'

**Comment by morchickit** [comment]
זה נגיד, יהיה מעולה, גם לכל המצב שח"כים חדשים נוספים / עוברים תפקיד

2016-11-23 19:01 GMT+00:00 Alonisser <notifications@github.com>:

**Comment by maryLoi** [comment]
האישיו הזה עדין פתוח? מה יש לעשות?

**Comment by morchickit** [comment]
כן, לוודא שבאמת כל חברי הוועדה במושב הזה בכנסת פתוחה מעודכנים כמו באתר הכנסת.
